deployment/docker-compose/pretix/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.6
+
+RUN apt-get update
+RUN apt-get install -y git libxml2-dev libxslt1-dev python-dev python-virtualenv locales \
+    libffi-dev build-essential python3-dev zlib1g-dev libssl-dev gettext libpq-dev \
+    libmysqlclient-dev libjpeg-dev
```

Comment:
`libmysqlclient-dev` cannot be found. In the referenced ubuntu version it is called `default-libmysqlclient-dev` for some reasons. Former and later versions call it `libmysqlclient-dev`.